src/Android.mk

```diff
@@ -24,60 +24,16 @@
 
 LOCAL_PATH:= $(call my-dir)
 
+include $(LOCAL_PATH)/Makefile.sources
+
 include $(CLEAR_VARS)
 
-LOCAL_SRC_FILES :=              \
-	dso_utils.c		\
-	gen6_mfc.c		\
-	gen6_mfc_common.c	\
-	gen6_mfd.c		\
-	gen6_vme.c		\
-	gen7_vme.c		\
-	gen7_mfc.c		\
-	gen7_mfd.c		\
-	gen75_mfd.c		\
-	gen75_mfc.c		\
-	gen8_mfc.c		\
-	gen8_mfd.c		\
-	gen8_vme.c		\
-	gen9_vme.c		\
-	gen9_mfc.c		\
-	gen9_mfc_hevc.c		\
-	gen9_mfd.c		\
-	gen75_picture_process.c	\
-	gen75_vme.c		\
-	gen75_vpp_gpe.c  	\
-	gen75_vpp_vebox.c	\
-	gen9_post_processing.c	\
-	i965_avc_bsd.c		\
-	i965_avc_hw_scoreboard.c\
-	i965_avc_ildb.c		\
-	i965_decoder_utils.c	\
-	i965_device_info.c	\
-	i965_drv_video.c	\
-	i965_encoder.c		\
-	i965_encoder_utils.c	\
-	i965_media.c		\
-	i965_media_h264.c	\
-	i965_media_mpeg2.c	\
-	i965_gpe_utils.c	\
-	i965_post_processing.c	\
-	gen8_post_processing.c	\
-	i965_render.c		\
-	i965_vpp_avs.c		\
-	gen8_render.c		\
-	gen9_render.c		\
-	intel_batchbuffer.c	\
-	intel_batchbuffer_dump.c\
-	intel_driver.c		\
-	intel_memman.c		\
-	object_heap.c		\
-	intel_media_common.c		\
-	$(NULL)
+LOCAL_SRC_FILES := $(source_c)
 
 LOCAL_MODULE := i965_drv_video
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := SHARED_LIBRARIES
+LOCAL_MODULE_RELATIVE_PATH := dri
```

Comment:
could you please put libva*.so and i965_drv_video.so to /vendor/lib folder, all vendor provided driver/hal should come to this folder.